src/core/deploy.ts

```diff
@@ -4,6 +4,7 @@ import debug from 'debug';
 import { API } from '../definition';
 import { BumpApi } from '../api';
 import { VersionRequest, VersionResponse } from '../api/models';
+import { Overlay } from '../core/overlay';
```

Comment:
The aliasing `as CoreOverlay` is necessary in the `command/overlay.ts` file because it also defines an `Overlay` names class. Here there's no naming collision, but if you think it makes the code clearer,  I'm fine with adding the alias here too :)